dist/templates/progress/_progress.dust

```diff
@@ -0,0 +1,32 @@
+{!}
+
+    Progress Component Template
+    ===========================
+    
+    This is the Progress component, which is used to display the gradual progress
+    a user makes through a multi-page workflow.
+    
+    For example, e-commerce checkout steps (bag, payment info, shipping info, 
+    review, confirmation, and so on).
+    
+    
+    How to Use
+    ----------
+    
+    1. Copy and Paste the below markup into your Adaptive.js project's
+       template directory
+    2. Add the template via includes in views or templates as needed
+    3. If relevant, it may be helpful to include in Almanac (Living Styleguide)
+
+{!}
+
+<div role="progressbar" aria-valuetext="Step {currentStep} of {maxStep}" aria-valuenow="{currentStep}">
```

Comment:
My intuition says the root node should have the root class. Looks like it was like that and got refactored out. Sounds like we need to discuss which oddness is worse: root class not on the root node, or root class with no styles. My feeling is root class should always be on the root node, and an empty class selector is fine here and there.